tests/tests.zig

```diff
@@ -17,4 +17,5 @@ comptime {
 
     // Language features
     _ = @import("language_features/cimport.zig");
+    // _ = @import("language_features/comptime_interpreter.zig");
```

Comment:
i think that having a useful collections of tests for your comptime interpreter would help other to also work on it.

Reply:
Oops forgot to reenable them, thanks for the headsup!